app/build.gradle

```diff
@@ -42,12 +42,13 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.android.support:appcompat-v7:27.0.2'
```

Comment:
I... am not sure why I needed to change this. Probably Android Studio was complaining. It's probably not needed, right?

I did check and I think the project has somewhat-outdated gradle and SDK versions... but I don't know if this change will be good.